### DISCUSSION TITLE
CXPatcher

### DISCUSSION THREAD

**Discussion by palmerj** [The-Wineskin-Project/WineskinServer#399] [General]
@Gcenx  I just saw CXPatcher being advertised on https://www.youtube.com/watch?v=ZeGh_u0Oa0I. code is here https://github.com/italomandara/CXPatcher. In Wineskin are we already getting these patches with the master version of DXVK?

**Comment by Gcenx** [comment]
That repo is just the patchers code along with the artifacts of DXVK-macOS, MoltenVK & ntdll.

The modified `ntdll` is from me and is something that’s already included within the CX22.1.1-rc builds.

The late wrapper update merged in the MoltenVK UE4 to tone mapping changes.

DXVK-macOS might be missing some of the tone mappings but I’ll eventually get around to it.

<br>

Honestly right now I’m happily playing Zelda: Tears Of The Kingdom on my switch.

**Reply by palmerj** [reply]
Awesome thank you once again. Enjoy Zelda!
